File: Java/Q94-Binary-Tree-Inorder-Traversal.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode() {}
 *     TreeNode(int val) { this.val = val; }
 *     TreeNode(int val, TreeNode left, TreeNode right) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */
class Solution {
  public static void help(TreeNode root,List<Integer> list){
      if(root==null){
          return;
      }else{
          help(root.left,list);
          list.add(root.val);
          help(root.right,list);
      }
  }
  public List<Integer> inorderTraversal(TreeNode root) {
      List<Integer> list=new ArrayList<Integer>();
      help(root,list);
      return list;
  }
}

File: Javascript/Q94-Binary-Tree-Inorder-Traversal.js
/**
 * Definition for a binary tree node.
 * function TreeNode(val, left, right) {
 *     this.val = (val===undefined ? 0 : val)
 *     this.left = (left===undefined ? null : left)
 *     this.right = (right===undefined ? null : right)
 * }
 */
/**
 * @param {TreeNode} root
 * @return {number[]}
 */

 function help(root,number){
  if(root==null){
      return;
  }else{
      help(root.left,number);
      number.push(root.val);
      help(root.right,number);
  }
}
var inorderTraversal = function(root) {
  var number=[];
  help(root,number);
  return number;
};